### ISSUE TITLE
Email Return-Path Setting is not used as Return-Path in Mail Transport

### ISSUE THREAD

**Issue by tkn98** [magento/magento2#8543]
Settings in the adminhtml backend regarding the outgoing mail messages return-path are ignored when Magento initiates the mail transport.

~~~
+-------------------------------+---------+----------+------------------+
| Path                          | Scope   | Scope-ID | Value            |
+-------------------------------+---------+----------+------------------+
| system/smtp/return_path_email | default | 0        | info@example.com |
| system/smtp/set_return_path   | default | 0        | 2                |
+-------------------------------+---------+----------+------------------+
~~~

In this case the `system/smtp/return_path_email` address "`info@exa.net`" is not used with outgoing transport.

The behavior is reproducible and was reproduced in Magento 2.1.3 and 2.1.4 clean instances.

* Reference: Magento Support Ticket #03200026

**Comment by tkn98** [comment]
Refs:

* #8420 / https://github.com/magento/magento2/commit/f5ac9f761ec81879d35a2295af12cb6f162b2499 (merge, not released, not a fix)

**Comment by magento-engcom-team** [comment]
Hi @tkn98 
We are closing thic ticket now as mentioned fix is already released with 2.2.0.
Thank you
